### PR TITLE
Speed improvements for pre-calculating returns

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/reporting/return.rb [hunk 1]
```diff
@@ -26,8 +26,9 @@ def populate!
     end
 
     def source_data(ids)
```
Comment (awisema) [inline app/models/reporting/return.rb]: Rename `ids` to be `enrollment_ids` to make it clear what is passed?
Maybe rename to `service_data` to go with `enrollment_data`?

File: app/models/reporting/return.rb [hunk 2]
```diff
@@ -26,8 +26,9 @@ def populate!
...
-      source_data_scope(ids).
-        order(client_id: :asc, service_history_enrollment_id: :asc, date: :asc)
+      GrdaWarehouse::ServiceHistoryService.where(service_history_enrollment_id: ids).
+        order(client_id: :asc, service_history_enrollment_id: :asc, date: :asc).
+        where(date: (Reporting::MonthlyReports::Base.lookback_start..Date.current))
     end
 
     private def source_data_scope(ids)
```
Comment (awisema) [inline app/models/reporting/return.rb]: Rename `ids`to `client_ids`?

File: app/models/reporting/return.rb [hunk 3]
```diff
@@ -38,6 +39,18 @@ def source_data(ids)
         where(date: (Reporting::MonthlyReports::Base.lookback_start..Date.current))
     end
 
+    def enrollment_data(ids)
```
Comment (awisema) [inline app/models/reporting/return.rb]: Rename `ids` to `client_ids`?

**Review by awisema — APPROVED** [review]

**Comment by eanders** [comment]
@awisema , went with `c_ids` instead of `client_ids` since that's a method in the model already.

**Pull request merged**